components/raftstore/src/store/fsm/store.rs

```diff
@@ -1771,6 +1803,9 @@ impl<EK: KvEngine, ER: RaftEngine> RaftBatchSystem<EK, ER> {
         let consistency_check_scheduler = workers
             .background_worker
             .start("consistency-check", consistency_check_runner);
+        let disk_check_scheduler = workers
+            .disk_check_worker
+            .start("disk-check-worker", disk_check_runner);
```

Comment:
How about spawn this runner in a background worker? It saves a thread.

Reply:
The reason why introducing a new worker is that reusing a background worker cannot limit the capacity of pending tasks.

Reply:
Another reason for using a new worker:
- We don't want other tasks to block the disk check runner, otherwise we would falsely assume the kv disk has an issue.

Reply:
Thx.

After discussed offline, the advice is accepted, which will use a background worker and limited capacity of `channel` for syncing tasks.